bleak/backends/bluezdbus/client.py

```diff
@@ -155,16 +155,17 @@ def on_value_changed(char_path: str, value: bytes) -> None:
 
         try:
             try:
-                async with async_timeout.timeout(timeout):
-                    reply = await self._bus.call(
-                        Message(
-                            destination=defs.BLUEZ_SERVICE,
-                            interface=defs.DEVICE_INTERFACE,
-                            path=self._device_path,
-                            member="Connect",
+                if not manager.is_connected(self._device_path):
```

Comment:
Let's add a comment explaining why we need this in the BlueZ backend (i.e. to make it behave more like other backends and to work around https://github.com/bluez/bluez/issues/89).

Reply:
a4ed7f8cfdea2752ed4ff14957ebf503be293d66